Ginger/GingerCoreCommon/Dictionaries/GingerDicser.cs

```diff
@@ -40,47 +50,37 @@ public enum eUserType
         Business
     }
 
-    public enum eTermResKey
-    {
-        BusinessFlow, BusinessFlows,
-        ActivitiesGroup, ActivitiesGroups,
-        Activity, Activities,
-        ConversionMechanism,
-        Variable, Variables,
-        RunSet, RunSets
-    }
-
     public class GingerDicser
     {
         public static string GetTermResValue(eTermResKey termResourceKey, string prefixString = "", string suffixString = "", bool setToUpperCase = false)
         {
             object termResValue = null;
+
             try
             {
-                // temp ugly code since we cannot load ResourceDic in GingerCoreNET - need another solution
-                if (termResourceKey == eTermResKey.Activity) termResValue = "Activity";
-                if (termResourceKey == eTermResKey.BusinessFlows) termResValue = "Business Flows";
-
-                if (termResValue == null)
-                {
-                    termResValue = "!!!termResValue!!!";  //FIXME
-                }
```

Comment:
you should use "AppReporter" for reporting the Error, reports are now supported also from GingerCoreCommon

Reply:
Used NewReporter class

Reply:
Code is merged with Master and then I was able to get the AppReporter class and used the same.